packages/create-plugin/templates/provisioning-template/README.md

```diff
@@ -0,0 +1,20 @@
+# Plugin name
+
+Brief description of the plugin.
+
+## Setup
+
+```bash
+docker-compose up -d
+```
+
+## Contact
+
+| Name | GitHub | Email |
+|------|--------|-------|
+|      |        |       |
+
+## Links
```

Comment:
Again this feels strange to ask for, especially as these are things I would expect to see in `links` in the plugin.json